osctiny/extensions/attributes.py

```diff
@@ -0,0 +1,73 @@
+"""
+Attributes extension
+--------------------
+
+.. versionadded:: 0.8.0
+"""
+import typing
+from urllib.parse import urljoin
+
+from lxml.objectify import ObjectifiedElement
+
+from ..utils.base import ExtensionBase
+
+
+class Attribute(ExtensionBase):
+    """
+    Access attribute namespaces and definitions
+    """
+    base_path = "/attribute/"
+
+    def list_namespaces(self) -> typing.List[str]:
+        """
+        Get a list of all namespaces
+
+        :return: List of namespace names
+        """
+        response = self.osc.request(
+            url=urljoin(self.osc.url, self.base_path),
+            method="GET"
+        )
+        content = self.osc.get_objectified_xml(response)
+        return [entry.get("name") for entry in content.findall("entry")]
+
+    def get_namespace_meta(self, namespace: str) -> ObjectifiedElement:
+        """
+        Get the meta of the namespace
+
+        :param namespace: namespace name
+        :return: Objectified XML element
+        """
+        response = self.osc.request(
+            url=urljoin(self.osc.url, f"{self.base_path}{namespace}/_meta"),
```

Comment:
On first read, I thought: "This must surely be a mistake! It should be like this:"
```suggestion
            url=urljoin(self.osc.url, f"{self.base_path}/{namespace}/_meta"),
```
"... but then why do the tests pass?"

Well, I understood very quickly, but it does "read wrong" to me, if you know what I mean.